### ISSUE TITLE
Inline listing of active endpoints in the console when starting the server

### ISSUE THREAD

**Issue by SylvainCorlay** [jupyter-server/fps#55]
When using jupyverse to serve a Voilà dashboard instead of a Jupyter notebook, we should be careful about not activating FPS extensions that would give too much privileges

Proposal:
 - upon starting an instance of Jupyverse, list all active routes grouped by FPS extension in a rich fashion from the OpenAPI spec.

This may prevent some unintended exposures of insecure end points.

Eventually, a tool for displaying OpenAPI specs in the console could be a separate package (RichAPI?) and a useful utility beyond jupyverse.

cc @davidbrochart @bollwyvl @adriendelsalle

**Comment by bollwyvl** [comment]
Yes: a performant, collapsible, nested, schema-aware treegrid would be
useful for so many things, both in the terminal in the browser!

It sounds less like a widget thing, perhaps, as so often these data sets
are useful operate at multiple levels of nesting... In this case, the
provider might be a single grouper level, but one could imagine the URL
route structure being arbitrarily nested.
